datastore/prometheus.go

```diff
@@ -0,0 +1,318 @@
+		"fromPod":            fromPod.Name,
+		"fromNamespace":      fromPod.Namespace,
+		"toService":          toSvc.Name,
+		"toNamespace":        toSvc.Namespace,
+		"toPod":              "",
+		"toHost":             "",
+		"toPort":             strconv.Itoa(int(req.ToPort)),
+		"status":             strconv.Itoa(int(req.StatusCode)),
+		"fromAcornProject":   fromPod.Labels[projectLabel],
+		"fromAcornApp":       fromPod.Labels[appLabel],
+		"fromAcornContainer": fromPod.Labels[containerLabel],
+		"fromAcornAccountId": fromPod.Labels[accountIdLabel],
+		"toAcornProject":     "",
+		"toAcornApp":         "",
+		"toAcornContainer":   "",
+		"toAcornAccountId":   "",
+	}).Inc()
+}
+
+func (p *PrometheusExporter) updateMetricsToOutbound(fromPod *PodEvent, req Request) {
+	p.latencyHistogram.With(prometheus.Labels{
+		"fromPod":            fromPod.Name,
+		"fromNamespace":      fromPod.Namespace,
+		"toHost":             req.ToUID, // req.ToUID is the website hostname in this case, i.e. google.com
+		"toService":          "",
+		"toPod":              "",
+		"toNamespace":        "",
+		"toPort":             strconv.Itoa(int(req.ToPort)),
+		"fromAcornProject":   fromPod.Labels[projectLabel],
+		"fromAcornApp":       fromPod.Labels[appLabel],
+		"fromAcornContainer": fromPod.Labels[containerLabel],
+		"fromAcornAccountId": fromPod.Labels[accountIdLabel],
+		"toAcornProject":     "",
+		"toAcornApp":         "",
+		"toAcornContainer":   "",
+		"toAcornAccountId":   "",
+	}).Observe(float64(req.Latency) / float64(1000000)) // divide by 1 million to convert nanoseconds to milliseconds
+
+	p.statusCounter.With(prometheus.Labels{
+		"fromPod":            fromPod.Name,
+		"fromNamespace":      fromPod.Namespace,
+		"toHost":             req.ToUID, // req.ToUID is the website hostname in this case, i.e. google.com
+		"toService":          "",
+		"toPod":              "",
+		"toNamespace":        "",
+		"toPort":             strconv.Itoa(int(req.ToPort)),
+		"status":             strconv.Itoa(int(req.StatusCode)),
+		"fromAcornProject":   fromPod.Labels[projectLabel],
+		"fromAcornApp":       fromPod.Labels[appLabel],
+		"fromAcornContainer": fromPod.Labels[containerLabel],
+		"fromAcornAccountId": fromPod.Labels[accountIdLabel],
+		"toAcornProject":     "",
+		"toAcornApp":         "",
+		"toAcornContainer":   "",
+		"toAcornAccountId":   "",
+	}).Inc()
+}
+
+func (p *PrometheusExporter) PersistRequest(request Request) error {
+	p.reqChanBuffer <- request
+	return nil
+}
+
+func (p *PrometheusExporter) PersistPod(pod Pod, eventType string) error {
+	podEvent := convertPodToPodEvent(pod, eventType)
+	p.podEventChan <- &podEvent
+	return nil
+}
+
+func (p *PrometheusExporter) PersistService(service Service, eventType string) error {
+	svcEvent := convertSvcToSvcEvent(service, eventType)
+	p.svcEventChan <- &svcEvent
+	return nil
+}
+
+func (p *PrometheusExporter) PersistDeployment(_ Deployment, _ string) error {
+	// ignore
+	return nil
+}
```

Comment:
I kept this and all of the other following `Persist` functions around so that the Prometheus Exporter can continue to implement the same interface that the old datastore backend did, even though we don't care about Deployments or any of these other events. So we just ignore them.